src/herumi/context.ts

```diff
@@ -6,25 +6,10 @@ type Bls = typeof bls;
 let blsGlobal: Bls | null = null;
 let blsGlobalPromise: Promise<void> | null = null;
 
-// Patch to fix multiVerify() calls on a browser with polyfilled NodeJS crypto
-declare global {
-  interface Window {
-    msCrypto: typeof window["crypto"];
-  }
-}
-
 export async function setupBls(): Promise<void> {
   if (!blsGlobal) {
     await bls.init(bls.BLS12_381);
 
-    // Patch to fix multiVerify() calls on a browser with polyfilled NodeJS crypto
-    if (typeof window === "object") {
-      const crypto = window.crypto || window.msCrypto;
-      // getRandomValues is not typed in `bls-eth-wasm` because it's not meant to be exposed
-      // @ts-ignore
-      bls.getRandomValues = (x) => crypto.getRandomValues(x);
```

Comment:
ahhh ok. so because the PR https://github.com/herumi/bls-eth-wasm/pull/23 exposed this?  really nice work on that PR by the way.  is super cool when we contribute to deps!! your rock sir!!